src/_listObject.scss

```diff
@@ -0,0 +1,226 @@
+
+@import "./listDefinitions";
+
+.#{$block} {
+  &--object {
+    .#{$block}__item {
+      @include padding-top-bottom(1rem);
+
+      height: 100%;
+      width: 100%;
+      box-sizing: border-box;
+      border-bottom: $fd-list-object-border-bottom;
+    }
+
+    .#{$block}-optional {
+      width: 100%;
+      padding-bottom: 0.75rem;
+
+      &__text {
+        @include fd-ellipsis();
+        @include fd-reset();
+
+        display: block;
+        width: 50%;
+        color: $fd-list-object-font-color;
+
+        @include fd-rtl() {
+          float: right;
+          width: 100%;
+          padding-bottom: 0.75rem;
+        }
+      }
+    }
+
+    .#{$block}__content {
+      flex-grow: 1;
+    }
+
+    .#{$block}__object {
+      width: 100%;
+
+      &--top-row {
+        display: flex;
+        vertical-align: top;
+      }
+
+      &--top-left {
+        flex: 3 3 10%;
+        width: 60%;
+
+        @include fd-rtl() {
+          text-align: right;
+        }
+      }
+
+      &--top-right {
+        width: 40%;
+        flex: 2 2 10%;
+        text-align: right;
+        padding-left: $fd-list-object-with-padding-value-one !important;
+        vertical-align: $fd-list-object-with-vertical-align-top !important;
+        margin-right: $fd-list-object-with-value-zero !important;
+
+        @include fd-rtl() {
+          text-align: left;
+          padding-right: $fd-list-object-with-padding-value-one !important;
+          padding-left: $fd-list-object-with-value-zero !important;
+          margin-left: $fd-list-object-with-value-zero !important;
+        }
+      }
+
+      &--bottom-row {
+        display: flex;
+        align-items: flex-end;
+        justify-content: flex-end;
+
+        > div:only-child {
+          padding: 0;
+        }
+      }
+
+      &--bottom-left {
+        flex: 3 3 10%;
+        text-align: left;
+        padding: 0.5rem 0.5rem 0.5rem 0;
+        display: inline-grid;
+
+        @include fd-rtl() {
+          text-align: right;
+          padding: 0.5rem 0 0.5rem 0.5rem;
+        }
+      }
+
+      &--bottom-right {
+        @include fd-ellipsis();
+
+        width: 50%;
+        max-width: 50%;
+        flex: 2 2 10%;
+        text-align: right;
+        margin-right: $fd-list-object-with-value-zero;
+        padding: 0.5rem 0 0.5rem 0.5rem;
+
+        @include fd-rtl() {
+          text-align: left;
+          margin-left: $fd-list-object-with-value-zero;
+          padding: 0.5rem 0.5rem 0.5rem 0;
+        }
+      }
+
+      &--bottom {
+        margin-top: 0.25rem;
+      }
+    }
+
+    .fd-object-number {
```

Comment:
I would prefer to avoid this pattern. We should add an additional class to `fd-object-number` and not use the same classes.

Reply:
updated, as suggested by Inna in below comments.